genie-proto/src/main/proto/genie.proto

```diff
@@ -31,3 +31,21 @@ message PongResponse {
   google.protobuf.Timestamp timestamp = 2;
   map<string, string> server_metadata = 3;
 }
+
+//
+service AgentRegistrationService {
```

Comment:
I thought we were going to go with a `GenieService`?

Reply:
There may be implications I'm not seeing and that are not coming out when a single service is present. Once we have 2 or 3 we can try to refactor into a single client and see what happens